index.js

```diff
@@ -73,7 +74,8 @@ function choo (opts) {
   function render (state, data, prev, name, createSend) {
     if (!_frame) {
       _frame = nanoraf(function (state, prev) {
-        const newTree = _router(state.location.pathname, state, prev)
```

Comment:
oh shit, you're right - I had the idea of passing in `state.location.href` and hash routing should be picked up by default now - could you perhaps give that a spin?